posts/2024-06-21-june-meetup/index.qmd

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 16:00:00 +0000
+categories: meetups
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+This month, we're shaking things up with an exciting line-up of lightning talks presented by you, our vibrant community members! This is your chance to share your knowledge, showcase your projects, and discuss your favourite R topics in a quick, 5-10 minute format.
+
```

Comment:
Normally the `<!--more-->` excerpt_separator would go after the first paragraph separating the overview from the body of the text. It means that only what comes before is summarised on the front page.

Very minor though and not something that needs changing.